compiler/AST/symbol.cpp

```diff
@@ -98,6 +98,15 @@ FnSymbol *gPrintModuleInitFn = NULL;
 FnSymbol* gChplHereAlloc = NULL;
 FnSymbol* gChplHereFree = NULL;
 FnSymbol* gChplDoDirectExecuteOn = NULL;
+FnSymbol *gGenericTupleTypeCtor = NULL;
+FnSymbol *gGenericTupleInit = NULL;
+FnSymbol *gGenericTupleDestroy = NULL;
```

Comment:
I don't see where this FnSymbol gets assigned anything other than NULL, even with a grepcomp.  I only see it get declared here, extern declared in symbol.h and used in tuples.cpp.